.github/workflows/integration-test.yaml

```diff
@@ -80,13 +80,10 @@ jobs:
           -u root -w /home/user/app \
           -v $PWD/data:/home/user/app/test_data \
           -v $PWD/sources:/home/user/app/sources \
-          -v $PWD/memory:/home/user/app/memory \
           ghcr.io/${{ github.repository_owner }}/${{ github.repository }}:pr-${{ github.event.pull_request.number }} \
-          bash -c "ollama serve & pip install pytest && pytest -W ignore::DeprecationWarning"
-      - uses: actions/cache/save@v4
-        with:
-          path: memory/
-          key: memory-${{ github.run_id }}-${{ github.run_attempt }}
+          bash -c "pip install pytest && pytest test/test_embed.py -W ignore::DeprecationWarning"
+        env:
+          OLLAMA_HOST: "https://totalsundae-ollama.hf.space"
```

Comment:
Rather than using the local running env's ollama, I instead use a free huggingface space that I am hosting. This space is very slow, so it doesn't really matter if I don't conceal it. It's public on my profile anyways.